specification/compute/resource-manager/Microsoft.Compute/stable/2021-07-01/compute.json

```diff
@@ -9610,10 +9613,30 @@
             "name": "VirtualMachineSizeTypes",
             "modelAsString": true
           }
+        },
+        "vmSizeProperties": {
+          "$ref": "#/definitions/VMSizeProperties",
+          "description": "Specifies the VM Size Properties for customizing VM sizes."
         }
       },
       "description": "Specifies the hardware settings for the virtual machine."
     },
+    "VMSizeProperties": {
+      "type": "object",
+      "properties": {
+        "vCPUsAvailable": {
+          "type": "integer",
+          "format": "int32",
```

Comment:
these properties are supposed to be nullable so pls. ensure they are reflected accordingly when generating the SDK.